README.md

```diff
@@ -17,6 +17,8 @@ A templated Django messaging library
     python manage.py migrate ilmoitin
     ```
 
+For Django 3 support, install the version `>=0.6`
```

Comment:
let's be even more explicit and say
```markdown
Use version 0.6.x for a project based on Django 3.x
Use version 0.5.x for a project based on Django 2.x
```
or something along these lines?

Reply:
Sounds good 🤔